### PR TITLE
Fix scheduled backup bug

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

fixes: #643 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test

Code changes

 - Has Go code change

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:
 
 ```release-note
fix scheduled backup bug: segmentation fault when backup user's password is empty
```

### CODE REVIEW

File: tests/cmd/e2e/main.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func main() {
 			TiKVImage:        fmt.Sprintf("pingcap/tikv:%s", initTidbVersion),
 			TiDBImage:        fmt.Sprintf("pingcap/tidb:%s", initTidbVersion),
 			StorageClassName: "local-storage",
-			Password:         "admin",
+			Password:         "",
```
Comment (xiaojingchen) [inline tests/cmd/e2e/main.go]: why change the `Password`

Comment (weekface) [inline tests/cmd/e2e/main.go]: NOTE: try to reproduce the error.

**Pull request closed without merging**